### ISSUE TITLE
Can't find library assimp on OS X 10.10

### ISSUE THREAD

**Issue by rcodddow** [ros-visualization/rviz#824]
Possibly needs a cmake module similar to Eigen and TinyXML

```
making version 1.11.4.
-- Assimp version has unified headers
-- checking for module 'yaml-cpp>=0.5'
--   package 'yaml-cpp>=0.5' not found
TODO: convert rviz tests to catkin.
-- libshiboken built for Release
Using SHIBOKEN_LIBRARY: /usr/local/Cellar/shiboken/1.2.2/lib/libshiboken-python2.7.dylib
-- Using default python: -python2.7
Using PYSIDE_LIBRARY: /usr/local/Cellar/pyside/1.2.2/lib/libpyside-python2.7.dylib
-- Found PythonLibs: /usr/lib/libpython2.7.dylib (found suitable version "2.7.6", minimum required is "2.7") 
-- Shiboken binding generator available.
CMake Warning at src/python_bindings/shiboken/CMakeLists.txt:14 (message):
  Shiboken version 1.2.2 would segfault when trying to process rviz (see
  https://bugreports.qt-project.org/browse/PYSIDE-218).  Therefore shiboken
  bindings are being skipped.


-- Found PythonInterp: /usr/local/bin/python (found suitable version "2.7.8", minimum required is "2.7") 
-- SIP binding generator available.
Python binding generators: sip
-- Configuring done
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   default_plugin
   rviz

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
-- Build files have been written to: /Users/robert/ros_catkin_ws/build_isolated/rviz
Linking CXX shared library /Users/robert/ros_catkin_ws/devel_isolated/rviz/lib/librviz.dylib
ld: library not found for -lassimp
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [/Users/robert/ros_catkin_ws/devel_isolated/rviz/lib/librviz.dylib] Error 1
make[1]: *** [src/rviz/CMakeFiles/rviz.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by scpeters** [comment]
Hasn't this been fixed by #825? I have 10.10 on my laptop, so I can test it there or on a virtual machine if testing is all that's needed.

**Comment by wjwwood** [comment]
Yes? I think so, thanks @scpeters.
